nevergrad/functions/images/core.py

```diff
@@ -10,6 +10,7 @@
 import torch.nn as nn
 import torch
 import torchvision
+from nevergrad.functions.images.imageLosses import SumAbsoluteDifferencesLoss, Koncept512Loss
```

Comment:
```suggestion
from . import imagelosses
```

Reply:
We actually also need to import Koncept512Loss but I'll do the relative import